specs/PushNotifications/PushNotifications-spec.md

```diff
@@ -2,120 +2,148 @@
+Desktop Bridge or SSP which may not satisfy 3rd party requirements.
 
-**Push Channels**: All the channel APIs are dependent on the caller app having a store provisioned identity.<br/>
-This means that Win32 and unpackaged apps that are not store distributed cannot use our Public APIs.<br/>
-Sideloaded UWPs have exactly the same problem. <br/>
+**Push Channels**: All the channel APIs are dependent on the caller app having a store provisioned
+identity. This means that Win32 and unpackaged apps that are not store distributed cannot use our
+Public APIs. Sideloaded UWPs have exactly the same problem.
 
-**Activation**: When the OS receives a Push over the WNS socket, it signals the Background <br/>
-Infrastructure to trigger a task and run some code on the app's behalf. This flow is undefined for Win32 apps <br/>
-that need to use Push today due to the limitations described above.
+**Activation**: When the OS receives a Push over the WNS socket, it signals the Background
+Infrastructure to trigger a task and run some code on the app's behalf. This flow is undefined for
+Win32 apps that need to use Push today due to the limitations described above.
 
-**Channel Request Error Handling**: The channel APIs exposed today can throw exceptions via their Projections <br/>
-Some errors can be critical errors while others are retryable errors. The msdn guidance for retrying isn't <br/>
-very clear and has been the root cause of multiple app bugs in the past. We want to abstract away the details of <br/>
-channel retry operations for developers in Windows App SDK.
+**Channel Request Error Handling**: The channel APIs exposed today can throw exceptions via their
+Projections Some errors can be critical errors while others are retryable errors. The msdn guidance
+for retrying isn't very clear and has been the root cause of multiple app bugs in the past. We want
+to abstract away the details of channel retry operations for developers in Windows App SDK.
 
 # Description
 
-At a high level, we need to provide a way for all Win32 applications to subscribe to and receive Push notifications <br/>
-irrespective of their app type. This includes unpackaged apps and packaged win32 (MSIX Desktop Bridge, Sparse Signed Packages). <br/>
-Moreover, all Push scenarios should adhere to OS resource management policies like Power Saver, Network Attribution, <br/>
-Enterprise group policies etc. The Windows App SDK Push component will abstract away the complexities of dealing with WNS <br/>
-channel registrations and Push related activations as much as possible freeing the developer to focus on other app <br/>
-related challenges.
+At a high level, we need to provide a way for all Win32 applications to subscribe to and receive
+Push notifications irrespective of their app type. This includes unpackaged apps and packaged win32
+(MSIX Desktop Bridge, Sparse Signed Packages). Moreover, all Push scenarios should adhere to OS
```

Comment:
You let out Win32App

...(MSIX Desktop Bridge, MSIX Win32App, Sparse Signed Packages)...

Win32App aka RuntimeBehavior=Win32App is like Desktop Bridge except instead of virtualized I/O and the like goodness, Win32App processes strongly favor (classic) Win32 compability. No Helium container, Loader has the classic 18+ mechanisms in the DLL Search Order vs 8, among other things.

P.S. Sparse Signed Packages isn't really another 'app type', more like a way of unpackaged apps opting in to MSIX goodness for runtime without replacing their installer with MSIX's. But sure, can't hurt to mention it here. All good.